pkg/storage/postgres/postgres.go

```diff
@@ -537,10 +537,20 @@ func (p *Postgres) ListStores(ctx context.Context, options storage.ListStoresOpt
 	ctx, span := tracer.Start(ctx, "postgres.ListStores")
 	defer span.End()
 
+	var whereClause sq.Sqlizer
+	if len(*options.IDs) > 0 {
```

Comment:
I'm curious, is there a specific reason why `options.IDs` is a pointer? I think `if len(options.IDs) > 0` and `sq.Eq{"id": options.IDs},` should also work, and we save ourselves from a potential nil pointer exception elsewhere in the codebase